distribution/lib/Standard/Base/0.0.0-dev/src/Data/Json/Extensions.enso

```diff
@@ -0,0 +1,194 @@
+import project.Any.Any
+import project.Data.Array.Array
+import project.Data.Json.Json
+import project.Data.Locale.Locale
+import project.Data.Map.Map
+import project.Data.Numbers.Number
+import project.Data.Range.Extensions
+import project.Data.Text.Text
+import project.Data.Vector.Vector
+import project.Error.Error
+import project.Meta
+import project.Nothing.Nothing
+
+from project.Data.Boolean import Boolean, True, False
+
+## ALIAS To JSON
+
+   Generically converts an atom into a JSON object.
+
+   The input atom is converted into a JSON object, with a `"type"` field set to
+   the atom's type name and all other fields serialized with their name as
+   object key and the value as the object value.
+
+   > Example
+     Convert a vector to JSON.
+         [1, 2, 3, 4].to_json
+Any.to_json : Json
```

Comment:
Moving these extension methods from `Json.enso` to `Json/Extensions.enso` helps with with imports somehow, right? How the imports looked and will look once this PR is integrated?

Reply:
I really like that we are getting a more explicit way to import the extension methods (by importing this separate module). It feels a lot cleaner (although I appreciate auto-importing them in the `from Standard.Base import all` also makes sense).